omnigibson/object_states/link_based_state_mixin.py

```diff
@@ -28,6 +29,14 @@ def link(self):
         """
         return self._default_link if self._link is None else self._link
 
+    @property
+    def links(self):
+        """
+        Returns:
+            None or RigidPrim: The link associated with this link-based state, if it exists
```

Comment:
Update docstring -- this is a dict now, right?

Reply:
done